test/jdl/jdl-importer.spec.js

```diff
@@ -504,6 +504,7 @@ describe('JDLImporter', () => {
                     ],
                     exportedApplications: [],
                     exportedDeployments: [],
+                    exportedApplicationsWithEntities: {},
```

Comment:
Why do we need it?

Reply:
I am moving save/formatting to the generator.
https://github.com/jhipster/generator-jhipster/issues/12178